go-controller/pkg/util/nicstobridge.go

```diff
@@ -382,3 +382,19 @@ func GetDPUHostInterface(bridgeName string) (string, error) {
 	// No host interface found in provided bridge
 	return "", fmt.Errorf("dpu host interface was not found for bridge %q", bridgeName)
 }
+
+func CreateLocalGatewayBridge() (string, error) {
+	bridge := "br-localgw"
```

Comment:
I know this is draft PR and hence not reviewing it per say, but are we really going back to our old architecture of creating yet another local-gateway bridge? I thought these changes are going to be only for MicroShift alone not for whole LGW mode?

We did a lot of effort in the past to bridge the gap between the two modes and allowing people to migrate between these modes realtime will less downtime, this seems like we are un-doing the work we did then?
I hope we won't add back the DGP? :)